Runtime/Scripts/GLTFSceneExporter.cs

```diff
@@ -1024,7 +1024,7 @@ private NodeId ExportNode(Transform nodeTransform)
 
 			if (ExportNames)
 			{
-				node.Name = nodeTransform.name;
+				node.Name = nodeTransform.name + _root.Nodes.Count;
```

Comment:
I dont quite understand how this change produces unique names.
Also i think i would prefer the name to be unchanged unless there is the need for it to be changed